.github/workflows/build-wasm_of_ocaml.yml

```diff
@@ -19,45 +17,44 @@ jobs:
         os:
           - ubuntu-latest
         ocaml-compiler:
-          - 4.14.x
-          - 5.0.x
-          - 5.1.x
-          - 5.2.x
+          - "4.14"
+          - "5.0"
+          - "5.1"
+          - "5.2"
           - ocaml-compiler.5.3.0~beta2
         separate_compilation:
           - true
         include:
           - os: macos-latest
-            ocaml-compiler: 5.2.x
+            ocaml-compiler: "5.2"
             separate_compilation: true
           - os: ubuntu-latest
-            ocaml-compiler: 4.14.x
+            ocaml-compiler: "4.14"
             separate_compilation: false
 
     runs-on: ${{ matrix.os }}
 
     steps:
-      - name: Set git to use LF
```

Comment:
This will need to come back once we start testing windows

Reply:
In the upcoming PR, I'm going to make changes to separate the jobs for testing and installation testing so that we can run all tests on all platforms.

Reply:
> separate the jobs for testing and installation testing

I don't understand what you mean by that.